client/src/modules/stock/lots/modals/edit.modal.js

```diff
@@ -0,0 +1,41 @@
+angular.module('bhima.controllers')
+  .controller('EditLotModalController', EditLotModalController);
+
+// dependencies injections
+EditLotModalController.$inject = [
+  'data', 'SessionService', 'LotService', 'NotifyService', '$uibModalInstance',
+];
+
+function EditLotModalController(Data, Session, Lots, Notify, Instance) {
+  const vm = this;
+  vm.model = {};
+  vm.enterprise = Session.enterprise;
+  vm.onDateChange = onDateChange;
+  vm.cancel = Instance.dismiss;
+  vm.submit = submit;
+
+  function startup() {
+    Lots.read(Data.uuid)
+      .then(lot => {
+        vm.model = lot;
+      })
+      .catch(Notify.handleError);
+  }
+
+  function onDateChange(date) {
+    if (date <= new Date()) { return; }
```

Comment:
I think we should let a user a set any date here - no need for this restriction.